spec/models/folder_spec.rb

```diff
@@ -141,39 +141,55 @@
     end
   end
 
-  describe 'elasticsearch' do
-    it { expect(described_class).to include(Elasticsearch::Model) }
-
-    # TODO, when we move to asynchronous indexing, remove this and replace with
-    # a test to ensure that jobs are created on create, update, delete
-    it { expect(described_class).to include(Elasticsearch::Model::Callbacks) }
+  describe 'creator' do
```

Comment:
Change to `describe '#creator' do`, we should try to follow the naming convention at http://betterspecs.org/#describe
So, `#` for instance methods and `::` for class methods. (I'm moving away from using the `.` as I've found that it's not explicit enough)